subgraphs/kwenta/protocols/kwenta/config/deployments/kwenta-optimism/configurations.json

```diff
@@ -0,0 +1,32 @@
+{
+  "deployment": "KWENTA_OPTIMISM",
+  "network": "optimism",
+  "marketManagerV1": [],
```

Comment:
Since there are no addresses for `marketManagerV1`, hence also no `FuturesV1Market`. 
If we are only tracking V2 markets, please mention this and why in the README